catalog/imaging/imaging-transformer-nitf/src/main/java/org/codice/alliance/transformer/nitf/common/NitfHeaderAttribute.java

```diff
@@ -154,29 +92,14 @@ private NitfHeaderAttribute(final String lName, final String sName,
         this.shortName = sName;
         this.longName = lName;
         this.accessorFunction = function;
-        this.attributeDescriptor = new AttributeDescriptorImpl(ATTRIBUTE_NAME_PREFIX + longName,
+        this.attributeDescriptor = new AttributeDescriptorImpl(longName,
```

Comment:
aren't all the attribute descriptors already created and available via blueprint?